guides/common/modules/con_remote-execution-in-projectname.adoc

```diff
@@ -0,0 +1,42 @@
+[id="Remote-Execution-in-{Project}_{context}"]
+= Remote Execution in {ProjectName}
+
+With remote execution, you can run jobs on hosts remotely from {SmartProxies} using shell scripts or Ansible tasks and playbooks.
+
+Use remote execution for the following benefits in {Project}:
+
+* Run jobs on multiple hosts at once.
+* Use variables in your commands for more granular control over the jobs you run.
+* Use host facts and parameters to populate the variable values.
+* Specify custom values for templates when you run the command.
+
+Communication for remote execution occurs through {SmartProxyServer}, which means that {ProjectServer} does not require direct access to the target host, and can scale to manage many hosts.
+ifeval::["{context}" == "ansible"]
+For more information, see xref:transport-modes-for-remote-execution_{context}[].
+endif::[]
+
+[NOTE]
+====
+Any {SmartProxyServer} base operating system is a client of the internal {SmartProxy} in the {ProjectServer}, and therefore this section applies to any type of host connected to {ProjectServer}, including {SmartProxies}.
+====
+
+To use remote execution, you must define a _job template_.
+A job template is a command that you want to apply to remote hosts.
+You can execute a job template multiple times.
+
+{Project} uses ERB syntax job templates.
+ifdef::managing-hosts[]
+For more information, see xref:Template_Writing_Reference_{context}[].
+endif::[]
+ifndef::managing-hosts[]
+For more information, see {ManagingHostsDocURL}Template_Writing_Reference_managing-hosts[Template Writing Reference] in _{ManagingHostsDocTitle}_.
+endif::[]
+
+Several job templates for shell scripts and Ansible are included by default.
```

Comment:
Unsure if we should use "provider_type" in some way. See `rg "provider_type:"` in foreman_remote_execution.

Reply:
Rerouted to https://github.com/theforeman/foreman-documentation/issues/2772